worlds/animal_well/client.py

```diff
@@ -462,7 +463,7 @@ def on_package(self, cmd: str, args: dict):
                         location_name = self.location_names.lookup_in_slot(location_id)
                         self.logic_tracker.check_logic_status[location_name] = CheckStatus.checked
             elif cmd == "RoomInfo":
-                pass
+                self.bean_patcher.save_seed = args["seed_name"]
```

Comment:
You should include the slot number in some fashion as well, so if someone is playing multiple Animal Well slots in the same session it won't put them all on the same save file.

Reply:
Oh, right, that might even be something I want to do by myself. If I have to cram more information into the save file name anyway, I might as well do the slightly more complicated hack to reserve a longer file name and see if it fixes the crash thing too.